api/services/ocean/spark/schemas/oceanSparkApplicationLoadBucket.yaml

```diff
@@ -0,0 +1,24 @@
+type: object
+required:
+- time
+- pending 
+- running
+description: >
+  Number of apps pending or running at a given point in time
+properties:
+  time:
+    type: string
+    format: date-time
+    description: >
+      Polling time
+    example: "2021-11-18T17:09:37+00:00"
+  pending:
+    type: number 
+    description: >
+      The number of apps successfully completed between >= start and < end
+    example: 0
+  running:
+    type: number
+    description: >
+      The number of apps that failed between >= start and < end
```

Comment:
Copy-pasting mistake. 

Number of applications that were running at the polling time.